src/components/UpgradePlanForm/UpgradePlanForm.js

```diff
@@ -2,6 +2,11 @@ import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import { InjectAppServices } from '../../services/pure-di';
 
+const fieldNames = {
```

Comment:
What do you think about it @fcoronelmakingsense? should I keep these constants for the names? or should I remove them?

Reply:
Any other consideration before merging?

Reply:
If we are going to use this type of consts for handling string constants in everywhere, we should agree on this, there are a lot of constants strings in doppler. In angularJS we used to have a constants file, here I'm not so sure of the best way to handle this. Perhaps we could inject constants? Keep in mind we will see this types again in the future surely @andresmoschini .

Reply:
I understand this constants are only for this model, ignore my previous comment , I will remove it. Will we handle string constants this way from now on? @fcoronelmakingsense @andresmoschini